data/campaigns/Descent_Into_Darkness/scenarios/05_Orc_War.cfg

```diff
@@ -32,7 +32,7 @@
             {STORYTXT_BACKGROUND travel.jpg}
         [/part]
         [part]
-            story=_ "Malin and Darken Volk continue their raids on the orcs and eventually force the usually fractious Whitefang chieftains to unite in opposition. The orcs pursue the two necromancers east toward the mouth of the river men call the Longlier and Elves call the Arkan-thoria, where they manage to surround the humans and force a fight."
+            story=_ "Malin and Darken Volk continue their raids on the orcs and eventually force the usually fractious Whitefang chieftains to unite in opposition. The orcs pursue the two necromancers east toward the mouth of the river men call the Longlier and elves call the Arkan-thoria, where they manage to surround the humans and force a fight."
```

Comment:
[The wiki](https://wiki.wesnoth.org/NotSpellingMistakes) says:

> Remember that we capitalize the name of a race when (and only when) referring to the entire race. Thus: "the lore of the Elves", but "a band of elves".

By that rule, shouldn't this be "Elves"?

Reply:
Ah, I missed that rule. I notice that, in general, writers seem to use capitals for non-human races, but use lower-case for referring to humans (seems rather inconsistent to me). Does Wesnoth use lower-case for humans? I note the description for woses uses lower-case.

If we are to follow this rule in the wiki, can we say '...mouth of the river Men call the Longlier and Elves call the Arkan-thoria...'?